db/migrations/20210111183822_get_block_heights_for_transformed_diffs.sql

```diff
@@ -0,0 +1,14 @@
+-- +goose Up
+
+CREATE OR REPLACE FUNCTION api.get_block_heights_for_transformed_diffs()
+    RETURNS SETOF BIGINT
+    LANGUAGE sql
+    STABLE
+AS
+$$
+SELECT MAX (block_height) FROM public.storage_diff WHERE status = 'transformed'
```

Comment:
Sorry to be super annoying but I think one cool thing to do here could be to actually execute this query against the db with `EXPLAIN ANALYZE` and then compare the results to `SELECT block_height FROM public.storage_diff WHERE status = 'transformed' ORDER BY block_height DESC LIMIT 1`.

Also, wondering if we want a separate PR/migration in vdb to add a partial index for diffs where status = transformed - could be worth doing that first to see if the index affects the explain analyze output.

Reply:
using MAX
```
Planning Time: 0.165 ms
Execution Time: 0.084 ms
```

using ORDER BY
```
Planning Time: 0.146 ms
Execution Time: 0.094 ms
```

Currently there are 3741377 `transformed` diffs in prod

Reply:
👍 🎉 

sounds like we may not need an index 😅